src/components/ContactPreview.tsx

```diff
@@ -0,0 +1,50 @@
+'use client'
+
```

Comment:
bug(unrelated to this MR): When I try to update my account just full name and username, without image, it fails (profiles gives error 400) and refreshes the page. Even with a picture if fails. The account I used is a.sandrina.p+1@gmail.com and the password is 1234567890.

Reply:
fixed!